packages/block-library/src/comment-template/edit.js

```diff
@@ -200,7 +210,7 @@ const CommentsList = ( {
 						activeComment={ activeComment }
 						setActiveComment={ setActiveComment }
 						blocks={ blocks }
-						firstComment={ comments[ 0 ] }
```

Comment:
We have to remove this and add a `firstComment` prop to the `CommentList` directly (one level higher in the tree). This is because otherwise, **at each level of comment nesting** we would create an active comment block! And this causes both the duplicate Inspector controls to show up and the weird selection issues.

You can compare it with [how this works](https://github.com/WordPress/gutenberg/blob/2323eeca3d0bc8b2723333ccb831ae731ae841aa/packages/block-library/src/post-template/edit.js#L211-L224) in the `post-template` block.